lib/rdstation/error_handler.rb

```diff
@@ -13,6 +13,8 @@ def raise_error
       raise error_class, array_of_errors.first if error_class < RDStation::Error
 
       error_class.new(array_of_errors).raise_error
+    rescue JSON::ParserError => error
+      raise error_class, {'error_message' => response.body}
```

Comment:
Missing spaces around the curly braces: `{ 'error_message' => response.body }`

Reply:
Instala o `linter-rubocop` nesse Atom aí! :D